frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Table } from 'react-bootstrap';
+import { ScheduledEventsTableRow } from '../ScheduledEventsTableRow/ScheduledEventsTableRow';
+import { ScheduledEvent } from '../../types/scheduledEvents';
+
+export const ScheduledEventsTable = (props: any) => {
```

Comment:
No unnecessary use of `any` ... give it an actual type.

Reply:
Also, give this function a type... `FunctionComponent`

Reply:
Unless u can't avoid it, listen to lint warnings immediately when they show up ... less of a hustle.